blocks/article-body-block/chains/article-body/_children/quote.jsx

```diff
@@ -32,23 +27,25 @@ export default ({ element, className }) => {
 		citation.content !== undefined &&
 		citation.content !== ""
 	) {
+		const citeClassName = classPrefix ? `${classPrefix}__citation` : null;
 		quoteItems.push(
 			// doesn't look like it has id
 			// via https://github.com/washingtonpost/ans-schema/search?p=2&q=citation&unscoped_q=citation
-			<PrimaryFont as="span" key={citation.content} className="citation-text">
+			<span key={citation.content} className={citeClassName}>
 				&mdash; &nbsp;
 				{citation.content}
-			</PrimaryFont>
+			</span>
 		);
 	}
 
 	return (
-		<blockquote
+		<Stack
+			as="blockquote"
```

Comment:
I think this is ok, but it seems like this would be an instance where Stack's "as" is being abused a little. If someone sets all of their stack styles, would they expect that to apply to text element like blockquote? Perhaps, continuing this, adding `<Paragraph as="blockquote"`

Reply:
couldn't we just use `<blockquote />` ?  Does it have to be a component?

Reply:
As the blockquote has two children - I used the stack to allow for consistent gap's to be applied, rather than having to resort back setting margin's everywhere, as if the blockquote does not have a citation the gap will account for that